test/integration/cookie_consent_integration_test.rb

```diff
@@ -164,6 +204,19 @@ class CookieConsentIntegrationTest < ActionDispatch::IntegrationTest
     end
   end
 
+  test 'custom analytics code present if only all cookies allowed' do
+    with_config_value(:require_cookie_consent, true) do
+      with_config_value(:custom_analytics_snippet_enabled, true) do
+        post cookies_consent_path, params: { allow: all_options }
+
+        get root_path
+
+        assert CookieConsent.new(cookies).allow_tracking?
+        assert_select '#custom-tracking-script', count: 1
```

Comment:
In this test I think you should set `custom_analytics_snippet` to something real, with real HTML elements, and then check if you can `assert_select` them - just to make sure they are not being sanitized/escaped.

Reply:
The default value for custom_analytics_snippet is now set to:
```html
<script id="custom-tracking-script"></script>
```